plugin/trino-firebolt/pom.xml

```diff
@@ -0,0 +1,189 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.trino</groupId>
+        <artifactId>trino-root</artifactId>
+        <version>419</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>trino-firebolt</artifactId>
+    <description>Trino - Firebolt</description>
+    <packaging>trino-plugin</packaging>
+
+    <properties>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+    </properties>
+
+    <repositories>
+        <repository>
+            <id>repsy</id>
+            <name>Firebolt Private Maven Repository on Repsy</name>
+            <url>https://repo.repsy.io/mvn/firebolt/maven-snapshots</url>
+        </repository>
+    </repositories>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.trino</groupId>
+            <artifactId>trino-base-jdbc</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>io.trino</groupId>
+            <artifactId>trino-plugin-toolkit</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>io.airlift</groupId>
+            <artifactId>configuration</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.firebolt</groupId>
+            <artifactId>firebolt-jdbc</artifactId>
+            <version>2.4.5-SNAPSHOT</version>
```

Comment:
Cant use a snapshot version.

Reply:
This snapshot contains a couple of fixes to the JDBC driver that's required in this change. A full version of the driver will be released shortly, this is added for the sake of creating this PR and getting some early feedback.
I will update to a full version before this PR is finalised.

Reply:
great

Reply:
Removed snapshot in favor of `2.4.6`.